.eslintrc.js

```diff
@@ -15,7 +18,38 @@ const config = {
                 noStrings: true
             }
         ]
-    }
+    },
+    overrides: [
+        {
+            files: ['**/*.gql.js'],
```

Comment:
We have some files named `*.gql.ce.js` and `.gql.ee.js` which I believe will get missed by this rule.

Reply:
Updated pattern in b9de346